zktest/test/java/org/zkoss/zktest/zats/test2/B86_ZK_4026Test.java

```diff
@@ -0,0 +1,34 @@
+/* B86_ZK_4026Test.java
+
+        Purpose:
+                
+        Description:
+                
+        History:
+                Wed Aug 15 10:48:23 CST 2018, Created by klyve
+
+Copyright (C) 2018 Potix Corporation. All Rights Reserved.
+*/
+package org.zkoss.zktest.zats.test2;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import org.zkoss.zats.mimic.ComponentAgent;
+import org.zkoss.zats.mimic.DesktopAgent;
+import org.zkoss.zktest.zats.ZATSTestCase;
+
+public class B86_ZK_4026Test extends ZATSTestCase {
+
+	@Test
+	public void testInnerWindowDetach() {
+		DesktopAgent da = connect();
+		ComponentAgent btn = da.query("#notifyBtn");
+		ComponentAgent lbl = da.query("#testLbl");
+		String uuid1 = lbl.getUuid();
+		btn.click();
+		String uuid2 = lbl.getUuid();
```

Comment:
You need to query `#testLbl` it again, because lbl is queried and stored.

Reply:
ok